npm install fails; git rev-parse (emerald-js-ui#feature/smart-contract)

Where `sandbox/emerald-starter-kit` contents created via `emerald new`:
```
~/sandbox/emerald-starter-kit master %
$ npm install
npm ERR! git rev-list -n1 feature/smart-contract: fatal: ambiguous argument 'feature/smart-contract': unknown revision or path not in the working tree.
npm ERR! git rev-list -n1 feature/smart-contract: Use '--' to separate paths from revisions, like this:
npm ERR! git rev-list -n1 feature/smart-contract: 'git <command> [<revision>...] -- [<file>...]'
npm ERR! git rev-list -n1 feature/smart-contract:
npm ERR! Linux 4.17.4-041704-generic
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "install"
npm ERR! node v8.10.0
npm ERR! npm  v3.5.2
npm ERR! code 128

npm ERR! Command failed: git rev-list -n1 feature/smart-contract
npm ERR! fatal: ambiguous argument 'feature/smart-contract': unknown revision or path not in the working tree.
npm ERR! Use '--' to separate paths from revisions, like this:
npm ERR! 'git <command> [<revision>...] -- [<file>...]'
npm ERR!
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /home/ia/sandbox/emerald-starter-kit/npm-debug.log
~/sandbox/emerald-starter-kit master %
$ head npm-debug.log
0 info it worked if it ends with ok
1 verbose cli [ '/usr/bin/node', '/usr/bin/npm', 'install' ]
2 info using npm@3.5.2
3 info using node@v8.10.0
4 silly loadCurrentTree Starting
5 silly install loadCurrentTree
6 silly install readLocalPackageData
7 silly install normalizeTree
8 silly loadCurrentTree Finishing
9 silly loadIdealTree Starting
~/sandbox/emerald-starter-kit master %
$ tail npm-debug.log
375 error node v8.10.0
376 error npm  v3.5.2
377 error code 128
378 error Command failed: git rev-list -n1 feature/smart-contract
378 error fatal: ambiguous argument 'feature/smart-contract': unknown revision or path not in the working tree.
378 error Use '--' to separate paths from revisions, like this:
378 error 'git <command> [<revision>...] -- [<file>...]'
379 error If you need help, you may report this error at:
379 error     <https://github.com/npm/npm/issues>
380 verbose exit [ 1, true ]
~/sandbox/emerald-starter-kit master %
$ cat npm-debug.log | grep smart-contract
66 silly fetchOtherPackageData emerald-js-ui@git+https://github.com/ETCDEVTeam/emerald-js-ui.git#feature/smart-contract
67 silly cache add args [ 'emerald-js-ui@git+https://github.com/ETCDEVTeam/emerald-js-ui.git#feature/smart-contract',
68 verbose cache add spec emerald-js-ui@git+https://github.com/ETCDEVTeam/emerald-js-ui.git#feature/smart-contract
69 silly cache add   raw: 'emerald-js-ui@git+https://github.com/ETCDEVTeam/emerald-js-ui.git#feature/smart-contract',
69 silly cache add   rawSpec: 'git+https://github.com/ETCDEVTeam/emerald-js-ui.git#feature/smart-contract',
69 silly cache add   spec: 'git+https://github.com/ETCDEVTeam/emerald-js-ui.git#feature/smart-contract',
69 silly cache add      ssh: 'git@github.com:ETCDEVTeam/emerald-js-ui.git#feature/smart-contract',
69 silly cache add      sshUrl: 'git+ssh://git@github.com/ETCDEVTeam/emerald-js-ui.git#feature/smart-contract',
69 silly cache add      httpsUrl: 'git+https://github.com/ETCDEVTeam/emerald-js-ui.git#feature/smart-contract',
69 silly cache add      gitUrl: 'git://github.com/ETCDEVTeam/emerald-js-ui.git#feature/smart-contract',
69 silly cache add      shortcut: 'github:ETCDEVTeam/emerald-js-ui#feature/smart-contract',
69 silly cache add      directUrl: 'https://raw.githubusercontent.com/ETCDEVTeam/emerald-js-ui/feature%2Fsmart-contract/package.json' } }
70 verbose addRemoteGit caching git+https://github.com/ETCDEVTeam/emerald-js-ui.git#feature/smart-contract
71 verbose addRemoteGit git+https://github.com/ETCDEVTeam/emerald-js-ui.git#feature/smart-contract is a repository hosted by github
72 silly tryClone cloning git+https://github.com/ETCDEVTeam/emerald-js-ui.git#feature/smart-contract via git+https://github.com/ETCDEVTeam/emerald-js-ui.git#feature/smart-contract
73 verbose tryClone git-https-github-com-ETCDEVTeam-emerald-js-ui-git-feature-smart-contract-83a7c8e1 not in flight; caching
130 silly validateExistingRemote git+https://github.com/ETCDEVTeam/emerald-js-ui.git#feature/smart-contract remote.origin.url: https://github.com/ETCDEVTeam/emerald-js-ui.git
131 verbose validateExistingRemote git+https://github.com/ETCDEVTeam/emerald-js-ui.git#feature/smart-contract is updating existing cached remote /home/ia/.npm/_git-remotes/git-https-github-com-ETCDEVTeam-emerald-js-ui-git-feature-smart-contract-83a7c8e1
361 verbose setPermissions git+https://github.com/ETCDEVTeam/emerald-js-ui.git#feature/smart-contract set permissions on /home/ia/.npm/_git-remotes/git-https-github-com-ETCDEVTeam-emerald-js-ui-git-feature-smart-contract-83a7c8e1
362 verbose resolveHead git+https://github.com/ETCDEVTeam/emerald-js-ui.git#feature/smart-contract original treeish: feature/smart-contract
363 info git [ 'rev-list', '-n1', 'feature/smart-contract' ]
364 error git rev-list -n1 feature/smart-contract: fatal: ambiguous argument 'feature/smart-contract': unknown revision or path not in the working tree.
364 error git rev-list -n1 feature/smart-contract: Use '--' to separate paths from revisions, like this:
364 error git rev-list -n1 feature/smart-contract: 'git <command> [<revision>...] -- [<file>...]'
365 silly fetchPackageMetaData Error: Command failed: git rev-list -n1 feature/smart-contract
365 silly fetchPackageMetaData fatal: ambiguous argument 'feature/smart-contract': unknown revision or path not in the working tree.
365 silly fetchPackageMetaData  error for emerald-js-ui@git+https://github.com/ETCDEVTeam/emerald-js-ui.git#feature/smart-contract { Error: Command failed: git rev-list -n1 feature/smart-contract
365 silly fetchPackageMetaData fatal: ambiguous argument 'feature/smart-contract': unknown revision or path not in the working tree.
365 silly fetchPackageMetaData   cmd: 'git rev-list -n1 feature/smart-contract' }
371 verbose stack Error: Command failed: git rev-list -n1 feature/smart-contract
371 verbose stack fatal: ambiguous argument 'feature/smart-contract': unknown revision or path not in the working tree.
378 error Command failed: git rev-list -n1 feature/smart-contract
378 error fatal: ambiguous argument 'feature/smart-contract': unknown revision or path not in the working tree.
```

[npm-debug.log](https://github.com/ETCDEVTeam/emerald/files/2397802/npm-debug.log)

--- a/downloaders/emerald-wallet.js
+++ b/downloaders/emerald-wallet.js
@@ -15,7 +15,6 @@ const p = path.resolve(__dirname, '../emerald-wallet');
 Git.Clone('https://github.com/ETCDEVTeam/emerald-wallet.git', p).then(() => {
   console.log('cloned');
   shell.cd(p);
-  shell.exec('git checkout feature/protocol-flow');
   shell.exec('npm install');
   shell.exec('npm run getemerald');
   shell.exec('npm run dist');

--- a/index.js
+++ b/index.js
@@ -10,7 +10,7 @@ prog
   .command('new', 'Create a new project')
   .action(async (args, options, logger) => {
     const dir = (args.length > 0) ? args[0] : process.cwd();
-    await truffleBox.unbox('https://github.com/BelfordZ/emerald-starter-kit.git', dir, {logger})
+    await truffleBox.unbox('https://github.com/ETCDEVTeam/emerald-starter-kit.git', dir, {logger})
     logger.info('New Emerald project created');
   })
 

